plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLDialectSchemaController.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.sql;
+
+import org.jkiss.code.NotNull;
+
+public interface SQLDialectSchemaController {
+    @NotNull
+    String getSchemaExistQuery(@NotNull String schemaName);
+
+    @NotNull
+    String getCreateSchemaQuery(
```

Comment:
I think we should move user name and password to optional parameters (e.g. Map<String, Object>). Because they are needed for Oracle only. Some other databases may need other parameters.